include/cocaine/rpc/asio/header.hpp

```diff
@@ -596,14 +649,17 @@ http2_integer_size(size_t sz, size_t bit_offset) {
 // Note dest should be at least 10 bytes long
```

Comment:
What can we assert here? function only takes a pointer and rely on the fact that it can fill at least 10 bytes in that piece of memory.